docs/BUILD.bazel

```diff
@@ -69,6 +79,41 @@ genrule(
     toolchains = ["@rules_sh//sh/posix:make_variables"],
 )
 
+[
+    genrule(
+        name = "readme_{}".format(rule),
+        srcs = [
+            "README_{}.md.tpl".format(rule),
+            "toolchains/{}.md".format(rule),
+        ],
+        outs = ["toolchains/{}/README.md".format(rule)],
+        # sorry for this mess. Bazel `str.format()` does not allow nested `{}`
+        # pairs and `awk` is sensitive to line breaks.
+        # TODO: consider alternative: leave reference for the end and just
+        # append the generated string with `cat`.
```

Comment:
I think one can escape `{` as `{{` and then it can be nested.

Reply:
🤔 Does that make *anything* substantially better? I'll try...

Reply:
It does not work, not even in Python 3.8 (`ValueError: unexpected '{' in field name`). This is a pathological case. Just look at the amount of additional braces! It is probably better to have a simpler setup in the first place, as noted in the comment.